src/main/resources/db/migration/V1__base.sql

```diff
@@ -0,0 +1,11 @@
+SET search_path = goal, pg_catalog;
```

Comment:
where is the schema ??

Always write migration code as rerunnable check how to use IF NOT EXISTS with create table and schema

Reply:
Done